src/components/Page.tsx

```diff
@@ -94,7 +94,14 @@ export function Page(props: {
   return isScreenFocused ? (
     <ScreenWrapper style={styles.container}>
       <View style={styles.titlePane}>
-        <Text style={[styles.title, {color: colors.text}]}>{props.title}</Text>
+        <View>
```

Comment:
do you need the extra view component here?

Reply:
Don't think so - edited